### DISCUSSION TITLE
permission denied

### DISCUSSION THREAD

**Discussion by mahamed-khattab** [community/community#37322] [New to GitHub]
When i begin the code in terminal it is a message say (permission denied)

**Comment by DSDmark** [comment]
Hi @mahamed-khattab,

> Add some more details for better understanding, for example:- screenshot or error.

About the error **permission denied** there are many following for caused that error:-

- You have used an incorrect email address in the GitHub SSH URL
- You have not configured your public SSH key in your GitHub account
- You must [create GitHub SSH keys](https://searchitoperations.techtarget.com/tutorial/Create-an-SSH-key-with-GitHub-for-network-access) to be used by the secure shell
